libs/sdk-core/src/breez_services.rs

```diff
@@ -1591,11 +1596,17 @@ impl BreezServices {
             )
             .await?;
 
+        if let Some(swap) = self.in_progress_swap().await? {
```

Comment:
It will only refresh the hook for the in progress (address that already has a transaction associated).
Aren't we messing the unused addresses?
Perhaps we should register for all addresses that return from `list_monitored` ?

Also perhaps we should only do this if the webhook url has changes?

Reply:
> perhaps we should only do this if the webhook url has changes?

Done: ca19d3d8a578262aa76846b767451fb8c458849a . Applies to all other callback types too.

Reply:
Yes, using `list_monitored` makes more sense.

Done in d6c2fe4e4af98c8fd62c21c9afd583d75ed640ee